src/content/en/fundamentals/instant-and-offline/get-started/httpcaching-6.md

```diff
@@ -0,0 +1,96 @@
+﻿project_path: /web/fundamentals/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+
+{# wf_updated_on: 2017-10-18 #}
+{# wf_published_on: 2017-10-18 #}
+
+# HTTP Caching {: .page-title }
+
+{% include "web/_shared/contributors/dgash.html" %}
+
+
+When someone visits a website, everything the site needs to display and operate has to come from somewhere. All the text, images, CSS styles, scripts, media files, and so on must be retrieved by the browser for display or execution. You can give the browser choices about where it can retrieve a resource from, and that can make a big difference in your page's load speed.
+
+The first time a browser loads a batch of page components, it stores them locally in a *browser cache*, a feature of modern browsers. The next time the browser hits that page, it can look in the cache for any previously-stored components and retrieve them from the cache faster than it can download them from the server. These conditions are called *empty cache* and *full cache*, although "full" as used here really means that the cache contains some or most -- but not necessarily all -- of the page components.
+
+While HTTP caching is standardized per the [Internet Engineering Task Force (IETF) specifications](https://tools.ietf.org/html/rfc7234), browsers may have multiple caches that differ in how they acquire, store, and retain content.
+
+Of course, every first-time visitor to your page arrives with an empty cache for that page. However, even repeat visitors may have an empty cache; they might have manually cleared it, or set their browser to do so automatically, or forced a fresh page load with a control-key combination. Some of the techniques we've already covered, such as combining and compressing resources, certainly benefit both empty- and full-cache users. 
+
+Still, a significant number of your users will revisit your site with at least some of its components already cached, and that can make a huge difference in load time. Maximizing cache usage is critical to speeding up return visits.
+
+## Enabling Caching
+
+Caching works by categorizing certain page resources in terms of how frequently or infrequently they change. Your site's logo image, for example, might almost never change, but your site's scripts might change every few days. It's beneficial to you and your users to determine which types of content are more static and which are more dynamic.
+
+It's also important to remember that what we think of as browser caching may in fact take place at any intermediate stop between the original server and the client-side browser, such as a proxy cache or a content delivery network (CDN) cache. 
+
+## Cache Headers
+
+Two main types of cache headers, *cache-control* and *expires*, define the caching characteristics for your resources. Typically, cache-control is considered a more modern and flexible approach than expires, but both headers can be used simultaneously.
```

Comment:
When we describe a concept, it would be great if we could inline link it straight out to the MDN docs for it. So for `cache-control` there's https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control.